customers_manager/customers/notification.py

```diff
@@ -0,0 +1,4 @@
+async def send_email(email_data: dict):
+    from main import send_email_task
```

Comment:
why local import? it should be global

Reply:
if it results in circular dependency error or something, you have to improve the project structure then